appium/tests/specs/mock/inbox/CheckArchieveAndMoveToInbox.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { MockApi } from 'api-mocks/mock';
+import { MockApiConfig } from 'api-mocks/mock-config';
+import {
+  SplashScreen,
+  SetupKeyScreen,
+  MailFolderScreen,
+  EmailScreen,
+  MenuBarScreen
+} from '../../../screenobjects/all-screens';
+
+describe('INBOX: ', () => {
+
+  it('user is able to archieve and move to inbox email', async () => {
+    const mockApi = new MockApi();
+    const testMessageSubject = 'Signed and encrypted message';
+
+    mockApi.fesConfig = MockApiConfig.defaultEnterpriseFesConfiguration;
+    mockApi.ekmConfig = MockApiConfig.defaultEnterpriseEkmConfiguration;
+    mockApi.addGoogleAccount('e2e.enterprise.test@flowcrypt.com', {
+      messages: [testMessageSubject, 'Signed only message'],
+    });
+
+    await mockApi.withMockedApis(async () => {
+      await SplashScreen.mockLogin();
+      await SetupKeyScreen.setPassPhrase();
+      await MailFolderScreen.checkInboxScreen();
+
+      // First find email and archieve it
+      await MailFolderScreen.clickOnEmailBySubject(testMessageSubject);
+      await EmailScreen.clickArchiveButton();
```

Comment:
@sosnovsky For mocked api, it throws error when user wants to archive thread. It returns updateLabels it not a function error.
Could you have a look at this as you have a better understanding of mocked apis?
```
[0-13] url:POST:/gmail/v1/users/me/threads/17d48c645be70702/modify?prettyPrint=false TypeError: DATA[this.acct].messages[index].updateLabels is not a function
[0-13]     at GoogleData.updateMessageLabels (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-data.ts:267:41)
[0-13]     at /gmail/v1/users/me/threads/? (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-endpoints.ts:154:14)
[0-13]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[0-13]     at async LoggedApi.Api.handleReq (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/lib/api.ts:141:30)
[0-13] 1ms | 500 POST /gmail/v1/users/me/threads/17d48c645be70702/modify?prettyPrint=false | {"mockError":"DATA[this.acct].messages[index].updateLabels is not a function","mockStack":"TypeError: DATA[this.acct].messages[index].updateLabels is not a function\n    at GoogleData.updateMessageLabels (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-data.ts:267:41)\n    at /gmail/v1/users/me/threads/? (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-endpoints.ts:154:14)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async LoggedApi.Api.handleReq (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/lib/api.ts:141:30)","noFormatter":"if you wanted a different error format, subclass from HttpErr and add a formatter"}
[0-13] url:POST:/gmail/v1/users/me/threads/17d48c645be70702/modify?prettyPrint=false TypeError: DATA[this.acct].messages[index].updateLabels is not a function
[0-13]     at GoogleData.updateMessageLabels (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-data.ts:267:41)
[0-13]     at /gmail/v1/users/me/threads/? (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/apis/google/google-endpoints.ts:154:14)
[0-13]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[0-13]     at async LoggedApi.Api.handleReq (/Users/mobile/Desktop/flowcrypt-ios/appium/api-mocks/lib/api.ts:141:30)

```

Reply:
@ioanmo226 yeah, I fixed it

Reply:
Great. Thanks

Reply:
Hmmm.. ui tests fail now.
Maybe because of new changes?